### DISCUSSION TITLE
2021-15: go-storage 的一次自我营销

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#220] [Announcements]
https://xuanwo.io/reports/2021-15/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by lisxray** [comment]
RSS是不是没有在更新呀

**Comment by Xuanwo** [comment]
周报系列的文章没有加入 RSS，你们会想在 RSS 推送里面看到吗？

**Comment by lisxray** [comment]
> @Xuanwo
> 周报系列的文章没有加入 RSS，你们会想在 RSS 推送里面看到吗？

原来如此，周报质量不错，还蛮有意思的

**Comment by izackwu** [comment]
> @Xuanwo
> 周报系列的文章没有加入 RSS，你们会想在 RSS 推送里面看到吗？

想~

**Comment by Xuanwo** [comment]
> > @Xuanwo
> > 周报系列的文章没有加入 RSS，你们会想在 RSS 推送里面看到吗？
> 
> 想~

加了~
